control/views.py

```diff
@@ -327,11 +326,25 @@ def get(self, request, *args, **kwargs):
         questionnaire = self.get_object()
         try:
             file = generate_response_file_list_in_xlsx(questionnaire)
-            # todo log action ?
+            self.add_log_entry(verb='xls file exported', questionnaire=questionnaire)
```

Comment:
on pourrait exporter autre chose en xls dans le futur (genre le questionnaire) qui ferait que le `verb` deviendrait pas clair. On doit pouvoir trouver un truc plus descriptif!

Par exemple 'exported responses in xls'  et 'exported responses in xls - fail' (pour qu'ils soient a coté dans l'ordre alphabetique, c'est pratique pour les trier dans la page des actions). C'est pas hyper beau, idees bienvenues :)

Reply:
Je trouve que c'est cool de faire:
- exported responses in xls
- exported responses in xls - fail

D'ailleurs, je me dis que le nom de la classe devrait être changé pour que se soit plus explicite, un truc du genre : `ExportXlsResponse`.